### ISSUE TITLE
Train/test split of Traffic4Cast dataset

### ISSUE THREAD

**Issue by morgankohler** [thuml/predrnn-pytorch#23]
Hi I am hoping to run some tests using the Traffic4Cast dataset tested in the paper but I need the train/test split that is used. Can this be provided? If downloading the most recent version of the competition, the Berlin data unzipped contains several files:

"BERLIN_map_high_res.h5  BERLIN_static.h5  BERLIN_test_additional_temporal.h5  BERLIN_test_temporal.h5  training"

Within the training folder there are 180 h5 files containing the frames for each day. Can you please let me know how you assign the train and test set so that I can accurately repeat your workflow? 

EDIT: Also, are you using the 9th channel (car accident report)? The 2021 data does not provide this channel.
